### ISSUE TITLE
Unable to publish ASP.NET Core Web Application

### ISSUE THREAD

**Issue by gimlichael** [aspnet/Tooling#956]
This i a known issue, as stated here: https://github.com/aspnet/Tooling/blob/eb3fc665816a4f2a22f9cba33317802ec814a5c5/known-issues-vs2017.md

However, I would like to know the status as it pains me to go back to Visual Studio 2015 just to re-write the code with the ability to publish.

What is the time horizon for a fix?

**Comment by mlorbetske** [comment]
@gimlichael @Spaceman1861 which publish issue(s) are you running in to?

cc @vijayrkn

**Comment by gimlichael** [comment]
Prior to your update, @mlorbetske, my issue was the dreadful: "DestinationFiles" refers to 1 item(s), and "SourceFiles" refers to 2 item(s). They must have the same number of items.", which forced my hand to do the same code in a VS 2015 project instead.

I have just tried again, with the latest VS 2017 RC update, and the problem appears to have been solved. However, I did not see this in the release notes?

Thanks.

**Comment by mlorbetske** [comment]
You're absolutely right, looking at the page now, we didn't mark it as fixed. @balachir could you help out with that please?
